server.ts

```diff
@@ -152,17 +153,20 @@ app.post("/conferences/:conferenceId/participants", async (req, res) => {
       const name = req.body.name;
       const phoneNumber = req.body.phoneNumber;
       console.log("name", name);
-      let participantId = await bandwidthRtc.createParticipant(conferenceId);
+      let participant: [string, string] = await bandwidthRtc.createParticipant(conferenceId);
+      let participantId = participant[0];
+      let deviceToken = participant[1];
```

Comment:
As I mentioned in the SDK changes, I think I would create or import an interface for `CreateParticipantResponse` so you don't have to use an array here. Then it could look something like:
```
const participant: CreateParticipantResponse = await bandwidthRtc.createParticipant(conferenceId);
const participantId = participant.participantId;
const deviceToken = participant.deviceToken;
```

I'm also not sure that we need to add `deviceToken` to the `Participant` interface, I don't see it getting used anywhere, we really just need to pass it to the device.

Reply:
That makes way more sense than what I was doing. Turns out we already had a CreateParticipantResponse, and I just didn't understand what it was doing, so I'm going to use that and flesh it out, thank you!